lib/apis/v3/felixconfig.go

```diff
@@ -246,6 +250,9 @@ type FelixConfigurationSpec struct {
 	// modes can use XDP. This is not recommended since it doesn't provide better performance than
 	// iptables. [Default: false]
 	GenericXDPEnabled *bool `json:"genericXDPEnabled,omitempty" confignamev1:"GenericXDPEnabled"`
+
+	// IPTablesBackend specifies which backend of iptables will be used. The default is legacy.
+	IPTablesBackend *IPTablesBackend `json:"iptablesBackend,omitempty" validate:"omitempty,iptablesBackend"`
```

Comment:
There are other iptables config params further up, I think this should be in the same "block" as those and it should follow the same casing.  For better or worse, the existing ones are "IptablesXXX"  (Think I followed the casing used by the iptables team, but I can't say it looks good!)